src/EPR.Calculator.Frontend/Controllers/CalculationRunNameController.cs

```diff
@@ -45,5 +82,32 @@ private static ErrorViewModel CreateErrorViewModel(string errorMessage)
                 ErrorMessage = errorMessage,
             };
         }
+
+        private async Task<HttpResponseMessage> GetHttpRequestAsync(string calculatorRunName)
```

Comment:
It's a Post Request. Method Name should be changed as per this.

Reply:
Good observation, changed to post - @Mazar-Shaik Done